docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.9'
+
+services:
+  symfony:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    container_name: symfony
+    restart: always
+    ports:
+      - 8080:80
+    volumes:
```

Comment:
Lute stores downloaded user term images in `public/userimages/`.  Will these volumes ensure that the images don't get accidentally deleted if someone destroys the containers and images?